SwiftGenStrings/SwiftTokenizer.swift

```diff
@@ -79,3 +103,17 @@ class SwiftTokenizer {
     }
 
 }
+
+private extension CharacterIterator {
+    
+    func skipWhitespace() {
+        while let current = current {
+            guard current == " " || current == "\t" || current == "\n" else {
```

Comment:
nit: Could string literals be substituted with the more generic `CharacterSet.whitespacesAndNewlines.contains`?